elizabeth/homework1.js

```diff
@@ -0,0 +1,173 @@
+/* Section 1
+a) True -- == is not exact and JS will auto convert "1" to a number.
+
+b) False -- === is exact, the values must match in type and value.
+
+c) True -- for the same reason "a" was true, both value and type are not checked, however it does not actually return a value.
+
+d) False -- the right side would be "undefined" and x!== undefined
+
+e) False -- empty string is "undefined" and the right side is a string equal to a space.
+
+f) True -- if x has a value of a string or a number, it is truthy, False if x has a value of Nan, empty string, or undefined.
+
+g) True -- x could equal undefined if it's value is ' '
+
+h) False -- string 9 and string 10 cannot be evaluated like numbers
+
+i) False -- typeof x will return a value, adding 1 to that will not ever return just "number"
+
+j) False -- same issue as "i" ^
+
+k) True -- now the typeof is on the expression (x % 2), JS will assume x is a number because of what follows it.
+
```

Comment:
What if x is set to be 'giraffe' or true? For this evaluation, most of the heavy lifting and conversion is being done by the % operator.